src/Workspaces/CSharp/Portable/Formatting/CSharpFormattingOptions.Parsers.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+
+namespace Microsoft.CodeAnalysis.CSharp.Formatting
+{
+    public static partial class CSharpFormattingOptions
+    {
+        internal static bool DetermindIfSpaceOptionIsSet(string value, SpacingWithinParenthesesOption parenthesesSpacingOption)
+            => (from v in value.Split(',').Select(v => v.Trim())
+                let option = ConvertToSpacingOption(v)
```

Comment:
You could just select ConvertToSpacingOption directly and the .Any() would just handle the null case for you, right?